tests/codeception/acceptance/Content/NewsCest.php

```diff
@@ -254,4 +255,39 @@ public function testMetaData(AcceptanceTester $I) {
     $I->assertEquals($values['featured_image_alt'], $I->grabAttributeFrom('meta[name="twitter:image:alt"]', 'content'), 'Metadata "twitter:image:alt" should match.');
   }
 
+  /**
+   * Test that default news variant displays correct fields.
+   */
+  public function testDefaultVariantFieldsDisplay(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/node/add/stanford_news');
+    $I->canSeeResponseCodeIs(200);
+    
+    // Select the default variant
+    $I->selectOption('#edit-su-news-variant', 'su_news_default');
+
+    // Verify that default variant fields are visible
+    $I->canSeeElement('#edit-su-news-dek-wrapper');
```

Comment:
Feedback from our 1:1

- When placeholder data is needed, use faker
- Test with `site-manager` or `contributor` roles
- Use field selector instead of id
- Use `canSeeInField()` instead of `canSeeElements()`
- Populate/fill in the fields
- Save
- Verify the information is reflected on the front-end

Scenario for functional
- Login for site manager
- Add a new News CT
- Verify that you cannot see a spotlight field (e.g. Quote)
- Select spotlight
- Verify that you can see the spotlight field (e.g. Quote)
- Fill in fields
- Save
- Verify information is reflected